helm/thingsboard/values.yaml

```diff
@@ -0,0 +1,285 @@
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  ssl:
+    enabled: false
+    secret: kubernetes-secret
+    cert: tls.crt
+    key: tls.key
+  port:
+    number: 8081
+    protocol: TCP
+    name: mqtt
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8883
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+http:
+  # kind can be either Deployment or StatefulSet
+  kind: StatefulSet
+  image:
+    repository: thingsboard/tb-http-transport
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8082
+    protocol: TCP
+    name: http
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8082
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+coap:
+  # kind can be either Deployment or StatefulSet
+  kind: StatefulSet
+  image:
+    repository: thingsboard/tb-coap-transport
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8083
+    protocol: UDP
+    name: coap
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8083
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+webui:
+  # kind can be either Deployment or StatefulSet
+  kind: Deployment
+  image:
+    repository: thingsboard/tb-web-ui
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8084
+    protocol: TCP
+    name: http
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8084
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+cassandraInitDB:
+  keyspace: thingsboard
+  replication:
+    class: SimpleStrategy
+    factor: 1
+
+cassandra:
+  # Database can be either postgres or hybrid (entities in postgres and timeseries in cassandra)
+  # for an postgresql database set cassandra.enabled as false
+  # for an hybrid database set cassandra.enabled as true
+  enabled: false
```

Comment:
Missing reference to `initDBConfigMap` defined in `templates/cassandra-init-db-configmap.yaml`?

Reply:
This was a previous discussion with @duhow ... to avoid using `{{ .Release.Name }}` in `values.yaml` which looks weird... we have replaced to a `set` of the value in the init configmap:
https://github.com/midokura/thingsboard-ce-k8s/blob/helm/helm/thingsboard/templates/cassandra-init-db-configmap.yaml#L17
It also kind of weird... but values which is what the final user consumer uses looks better. Not a strong opinion on this. I can just say that like done in this PR, it's currently working

Reply:
Yeah, both are weird, but I find https://github.com/midokura/thingsboard-ce-k8s/blob/helm/helm/thingsboard/templates/cassandra-init-db-configmap.yaml#L17 more confusing than simply setting it in `values.yml`. IMHO, things like that belong in `_helpers.tpl`.